daemon/algod/api/server/v1/handlers/handlers.go

```diff
@@ -323,6 +323,15 @@ func AccountInformation(ctx lib.ReqContext, w http.ResponseWriter, r *http.Reque
 		return
 	}
 
+	apiParticipation := v1.Participation{
+		Round:           uint64(lastRound), // these fields are redundant when embedded in Account, but enables reuse of the model elsewhere.
+		Address:         addr.String(),
+		ParticipationPK: record.VoteID[:],
+		VRFPK:           record.SelectionID[:],
+		VoteFirst:       uint64(record.VoteFirstValid),
+		VoteLast:        uint64(record.VoteLastValid),
+		VoteKeyDilution: uint64(record.VoteKeyDilution),
```

Comment:
`VoteKeyDilution` is already uint64, so no need to cast it

Reply:
fixed in #163!